test/services/post-install-utility.test.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-console */
+const tmp = require('tmp');
+const chalk = require('chalk');
+const { expect, test, constants } = require('@twilio/cli-test');
+const sinon = require('sinon');
+const { ConfigData } = require('@twilio/cli-core').services.config;
+
+const { PostInstallDisplayManager, PORT_WARNING } = require('../../src/services/post-install-utility');
+
+describe('services', () => {
+  describe('post-install-utility', () => {
+    beforeEach(() => {
+      tempConfigDir = tmp.dirSync({ unsafeCleanup: true });
+      warnSpy = sinon.spy(console, 'warn');
+      logSpy = sinon.spy(console, 'log');
+    });
+
+    afterEach(() => {
+      tempConfigDir.removeCallback();
+      warnSpy.restore();
+      logSpy.restore();
+    });
+
+    test.it('should display grid when no profiles or projects configured', () => {
+      const configData = new ConfigData();
+      const displayManager = new PostInstallDisplayManager(tempConfigDir.name, configData);
+
+      displayManager.displayMessage();
+      expect(displayManager.userConfig).to.not.be.undefined;
+      expect(console.log.calledWith('* To get started, please create a twilio-cli profile:                   *')).to.be
+        .true;
+      expect(console.log.calledWith('*     twilio profiles:create                                            *')).to.be
+        .true;
+      expect(console.log.calledWith('*     OR                                                                *')).to.be
+        .true;
+      expect(console.log.calledWith('*     twilio login                                                      *')).to.be
```

Comment:
Recommend these not be so specific. I.e., if the spacing in the code changes that tests also have to be updated. It's just more maintenance.

Reply:
Makes sense, changed at all places!